packages/forge/LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) [year] [fullname]
+Copyright (c) 2025 Umut Sirin
```

Comment:
Sirin dogru yazilmamis

Reply:
I used https://github.com/usirin/monorepo/blob/main/LICENSE as a reference for this change.